vite.config.ts

```diff
@@ -8,4 +8,9 @@ export default defineConfig({
 		host: "0.0.0.0",
 		port: 3000,
 	},
+	build: {
+		rollupOptions: {
+			external: ["/sdk/v2/browser-sdk.esm.js", "/sdk/v1/iamport.esm.js"],
+		},
+	},
```

Comment:
이거 이렇게 설정하기보다는 개별 import에 [vite-ignore](https://github.com/yusukebe/hono-playground/blob/9f025863318052f081fb5a2a94110d5fccfae8ac/index.html#L59) 추가하는 게 더 관리하기 편할 것 같아요

Reply:
아 이런게 있었군요

Reply:
음... 오류가 나는군요. 일단 이렇게 해두라고 권장하고 있어서 그대로 해야될 것 같습니당..

```shell
error during build:
Error: [vite]: Rollup failed to resolve import "/sdk/v1/iamport.esm.js" from "/home/lastation/portone/browser-sdk/playground/src/state/v1.ts".
This is most likely unintended because it can break your application at runtime.
If you do want to externalize this module explicitly add it to
`build.rollupOptions.external`
```

Reply:
음 아마 vite-ignore가 제대로 안 먹은 것 같은데 삼항연산자라서 어디에 붙이느냐에 따라 동작이 달라질 것 같긴 하네요.... 명확하게 하려면 external 쓰는 게 나을 것 같아 보이는

Reply:
혹시나 해서 따로 `VITE_BROWSER_SDK_V2=/sdk/v2/browser-sdk.esm.js` 형태로 주소를 분리하고
```ts
const { default: PortOne, slots } = import.meta.env.VITE_BROWSER_SDK_V2
  ? await import(/* @vite-ignore */ import.meta.env.VITE_BROWSER_SDK_V2)
  : await import("https://cdn.portone.io/v2/browser-sdk.esm.js");
```
이렇게 사용해봤는데, vite-ignore를 안 쓰면 경고가 뜨는건 사라져도 오류 자체는 동일하게 뜨더라구요 😢 
그래서 이런식으로 정리해봤습니다

```ts
build: {
  rollupOptions: {
    external: [env.VITE_BROWSER_SDK_V1, env.VITE_BROWSER_SDK_V2],
  },
},
```